raiden-dapp/src/utils/query-params.ts

```diff
@@ -1,4 +1,18 @@
 import AddressUtils from '@/utils/address-utils';
+import { BigNumber, bigNumberify } from 'ethers/utils';
+
+export function getPaymentId(queryParam: any): BigNumber | undefined {
+  let paymentId = undefined;
+  if (queryParam && typeof queryParam === 'string') {
+    try {
+      paymentId = bigNumberify(queryParam);
+    } catch (error) {
+      return undefined;
+    }
+  }
+
+  return paymentId;
+}
```

Comment:
Nitpick:
This function isn't doing much. Consider to simplify it.

```suggestion
export function getPaymentId(queryParam: any): BigNumber | undefined {
  try {
    return bigNumberify(queryParam);
  } catch (_error) {
    return undefined;
  }
}
```